docs/tutorials/tutorial_flask_ask_python.md

```diff
@@ -77,7 +77,7 @@ INFO  2016-09-12T17:34:52.202Z Connected - proxy.bespoken.tools:5000
 Since tidepooler.py does not have a path it is listening on, simply `/`, you can remove the `YOUR/SKILL/PATH` from the provided URL and copy and paste as your endpoint:
```

Comment:
This lines no longer match the functionality since the url no longer includes the actual path